pravega-sensor-collector/src/test/java/io/pravega/sensor/collector/PravegaSensorCollectorIntegrationTests.java

```diff
@@ -60,9 +69,91 @@ public void tearDown() {
 
     }
 
+    @Test
+    @Timeout(value = 3, unit = TimeUnit.MINUTES)
+    @Order(1)
+    public void testPSCDataIntegration() {
+        try {
+            copyHelloWorldFile();
+        } catch (IOException e) {
+            throw new RuntimeException(e);
+        }
+
+        final DeviceDriverManager deviceDriverManager = new DeviceDriverManager(properties);
+        Service startService = deviceDriverManager.startAsync();
+        try {
+            startService.awaitRunning(Duration.ofSeconds(30));
+            Thread.sleep(25000);
```

Comment:
Is this sleep necessary? Why 25K?

Reply:
reduced sleep interval to 12-15 seconds from 25 seconds.